dashboard/src/components/Config/PkgRepoList/PkgRepoForm.tsx

```diff
@@ -0,0 +1,1701 @@
+                      <label htmlFor="kubeapps-repo-filter-regex">Regular Expression</label>
+                      <CdsControlMessage>
+                        Mark this box to treat the filter as a regular expression
+                      </CdsControlMessage>
+                      <input
+                        id="kubeapps-repo-filter-regex"
+                        type="checkbox"
+                        onChange={handleFilterRegexChange}
+                        checked={filterRegex}
+                      />
+                    </CdsCheckbox>
+                  </>
+                )}
+              </CdsFormGroup>
+            </CdsAccordionContent>
+          </CdsAccordionPanel>
+          <CdsAccordionPanel id="panel-advanced" expanded={accordion[3]}>
+            <CdsAccordionHeader onClick={() => toggleAccordion(3)}>Advanced</CdsAccordionHeader>
+            <CdsAccordionContent>
+              <CdsFormGroup layout="vertical">
+                {plugin?.name !== PluginNames.PACKAGES_HELM && (
+                  <CdsInput>
+                    <label htmlFor="kubeapps-repo-interval">Synchronization Interval</label>
+                    <input
+                      id="kubeapps-repo-interval"
+                      type="text"
+                      placeholder="10m"
+                      value={interval}
+                      onChange={handleIntervalChange}
+                    />
+                    <CdsControlMessage>
+                      Time (expressed as a{" "}
+                      <a
+                        href={"https://pkg.go.dev/time#ParseDuration"}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        Golang duration
+                      </a>
+                      ) to wait between synchronizing the repository.
+                    </CdsControlMessage>
+                  </CdsInput>
+                )}
+                {plugin?.name === PluginNames.PACKAGES_HELM && (
+                  <CdsCheckbox>
+                    <label htmlFor="kubeapps-repo-performvalidation">Perform Validation</label>
+                    <CdsControlMessage>
+                      Ensure that a connection can be established with the repository before adding
+                      it.
+                    </CdsControlMessage>
+                    <input
+                      id="kubeapps-repo-performvalidation"
+                      type="checkbox"
+                      onChange={handlePerformValidationChange}
+                      checked={performValidation}
+                    />
+                  </CdsCheckbox>
+                )}
+
+                <CdsToggleGroup>
+                  <CdsToggle>
+                    <label>
+                      {isUserManagedCASecret
+                        ? "Use user-managed secrets"
+                        : "Use Kubeapps-managed secrets"}
```

Comment:
Just throwing the idea: what about using more friendly label `I will manage the secrets`, `Let Kubeapps manage the secrets`, or similar?

Reply:
Yeah, those "Provide the secret values" and "Use existing secrets" sound good too!